pkgs/tools/networking/mu/default.nix

```diff
@@ -1,31 +1,26 @@
 { stdenv, fetchFromGitHub, sqlite, pkgconfig, autoreconfHook, pmccabe
-, xapian, glib, gmime, texinfo , emacs, guile
+, xapian, glib, gmime3, texinfo , emacs, guile
 , gtk3, webkitgtk24x-gtk3, libsoup, icu
 , withMug ? false }:
 
 stdenv.mkDerivation rec {
   name = "mu-${version}";
-  version = "1.0";
+  version = "1.2";
 
   src = fetchFromGitHub {
     owner  = "djcb";
     repo   = "mu";
-    rev    = "v${version}";
-    sha256 = "0y6azhcmqdx46a9gi7mn8v8p0mhfx2anjm5rj7i69kbr6j8imlbc";
+    rev    = version;
+    sha256 = "0yhjlj0z23jw3cf2wfnl98y8q6gikvmhkb8vdm87bd7jw0bdnrfz";
   };
 
-  # 0.9.18 and 1.0 have 2 failing tests but previously we had no tests
-  patches = [
-    ./failing_tests.patch
```

Comment:
If we no longer need this, please remove the file as well.

Reply:
Indeed. Done, thanks!